baseapp/abci.go

```diff
@@ -148,6 +151,147 @@ func (app *BaseApp) FilterPeerByID(info string) abci.ResponseQuery {
 	return abci.ResponseQuery{}
 }
 
+func (app *BaseApp) GetAppHash(req abci.RequestGetAppHash) (res abci.ResponseGetAppHash) {
+	cms := app.cms.(*rootmulti.Store)
+
+	appHash, err := cms.GetAppHash()
+	if err != nil {
+		panic(err)
```

Comment:
Is calling panic a standard pulled from the cosmos sdk?

Reply:
Yes, for `ABCI` methods, if something goes wrong, there's no way to gracefully throw an error other than `panic`. If you look at the `BeginBlock` implementation, you'll see `panic` calls.